src/Domain/Insights/Syntax.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Domain\Insights;
+
+use NunoMaduro\PhpInsights\Domain\Contracts\GlobalInsight;
+use NunoMaduro\PhpInsights\Domain\Contracts\HasDetails;
+use NunoMaduro\PhpInsights\Domain\Details;
+use PHP_CodeSniffer\Config;
+use Symfony\Component\Process\Process;
+
+/**
+ * @internal
+ */
+final class Syntax extends Insight implements HasDetails, GlobalInsight
```

Comment:
Can we name it `SyntaxChecker` or something a little more descriptive

Reply:
I renamed it SyntaxCheck :wink: